test/onnx/test_fx_to_onnx_with_onnxruntime.py

```diff
@@ -146,6 +147,25 @@ def func(x, b=torch.tensor(1.0)):
             func, (tensor_x,), b=torch.tensor(5.0)
         )
 
+    # beartype.roar.BeartypeCallHintParamViolation:
+    # @beartyped onnxscript.function_libs.torch_aten.graph_building.TorchScriptGraph.add_input()
```

Comment:
Is `8.0` an expected input to the model? Should it not be dynamic?

Reply:
@titaiwangms I wonder if we should handle constants as model inputs

Reply:
For `_run_test_with_fx_to_onnx_exporter_and_onnx_runtime(func, (tensor_x, 8.0))` this call, yes it is expected as input.

Reply:
@justinchuby if fx has it, we should addit. LMK if you guys ever find annotation of node.meta["val"]